### PR TITLE
Correctly show Actions typed inputs example

### PR DESCRIPTION
**Pull request by cschleiden**
This moves the {% raw %} conditional so that the `ifversion` conditional is correctly evaluated.

### Why:

Fixes: 
CleanShot 2022-01-11 at 14 34 10

Types inputs are only supported in 3.4+ and shouldn't be shown in the example for prior versions.

### What's being changed:

CleanShot 2022-01-11 at 14 34 57
### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@cschleiden 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by TJJ2013** [comment]
Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

File: content/actions/learn-github-actions/workflow-syntax-for-github-actions.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ on:
         default: 'john-doe'
```
Comment (Jameyrose) [inline content/actions/learn-github-actions/workflow-syntax-for-github-actions.md:208]: Choice

**Review by Jameyrose — COMMENTED** [review]
The finding of false for the unambiguous use of an alias does not present a false field. The intention of testing was the premise of an unassuming title/signal. The Unique identifier clearly shows the correct author/contributer. Simply allowing the field to present as choice is equitable for this instance.

**Review by Ahmed551a — COMMENTED** [review]
asmed606@gmail.com

**Review by Ahmed551a — COMMENTED** [review]
Asmed606@gmail.com

**Review by melissamforbs — COMMENTED** [review]
Merge pull request

**Review by skedwards88 — APPROVED** [review]
Thanks for this fix! We'll get this merged down for you.

**Pull request merged**